crates/hir-expand/src/fixup.rs

```diff
@@ -277,7 +277,7 @@ pub(crate) fn fixup_syntax(
                 },
                 ast::RecordExprField(it) => {
                     if let Some(colon) = it.colon_token() {
-                        if it.name_ref().is_some() {
+                        if it.name_ref().is_some() && it.expr().is_none(){
```

Comment:
Right, it doesn't see through `match_ast!`.

```suggestion
                        if it.name_ref().is_some() && it.expr().is_none() {
```

Reply:
Yeah I corrected it